project/profiles/capacity4more/modules/c4m/og/c4m_og/c4m_og.module

```diff
@@ -108,6 +108,181 @@ function c4m_og_form_alter(&$form, $form_state) {
 }
 
 /**
+ * Implements hook_menu().
+ */
+function c4m_og_menu() {
+  $items['group/%/%/admin/owner'] = array(
+    'page callback' => 'drupal_get_form',
+    'page arguments' => array('c4m_og_replace_owner_build_form', 1, 2),
+    'access callback' => 'c4m_og_user_is_owner',
+    'access arguments' => array(1,2),
+    'title' => t('Replace Owner'),
+  );
+
+  return $items;
+}
+
+/**
+ * Form builder; OG manager change page.
+ */
+function c4m_og_replace_owner_build_form($form, $form_state, $entity_type, $etid) {
```

Comment:
Change `$etid` to `$entity_type_id` (If that's what it means).
Add params and return to doc.